### ISSUE TITLE
Extreme performance drop in a larger modpack.

### ISSUE THREAD

**Issue by Noxalas** [LakMoore/Smooth-Entity-Light#9]
I love SEL because it's everything I ever wanted from a dynamic lights-like mod, but recently I've seen a giant drop in performance. It worked perfectly before on my 140+ modpack, but after a couple of updates my fps suddenly drop from 300-500 to 4-60.

I've tried disabling a ton of mods, but nothing helps. I've done tests where SEL didn't even have any of the modules enabled, and even then it was unplayable.

Any idea what would be causing this?

**Comment by ghost** [comment]
Yes i have this problem too. When i create my modpack with 150 mods its work fine, but after i add 200+ mod i get fps drop from 4 to 80. I disabled this mod and use Optifine native dynamic ligths. All works good and fps dont drop with setting Dynamic Lights: Fancy. Its stability all time when i play everywhere.

**Comment by strobili** [comment]
Same experience but with a modpack with around 30 mods.
